ksqldb-parser/src/main/java/io/confluent/ksql/util/QueryMask.java

```diff
@@ -57,11 +42,13 @@ private String getMaskedStatement(final String query) {
     }
   }
 
-  private String maskConnectProperties(final String query) {
+  private static String maskConnectProperties(final String query) {
     /*
       Failed to parse statement. Mask disallowed keys by string replacing. This is a best effort
       attempt. The assumption is that properties are in right format:
-      "(\"[^\"]+\")\\s*=\\s*('[^']*'|\"[^\"]*\")")
+      "(\"[^\"]+\")\\s*=\\s*('[^']*'|\"[^\"]*\")") i.e. it matches key wrapped in double quotes
```

Comment:
The key should only ever be single-quoted, for valid statements: https://github.com/confluentinc/ksql/blob/b6fc8d91152db35c2f8067f4db44ed6cba1c172b/ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4#L597